test/System.Net.Http.Formatting.Test/Formatting/BsonMediaTypeFormatterTests.cs

```diff
@@ -262,10 +262,10 @@ public async Task FormatterThrowsOnWriteWithInvalidContent()
             BsonMediaTypeFormatter formatter = new BsonMediaTypeFormatter();
             HttpContent content = new StringContent(String.Empty);
             MemoryStream stream = new MemoryStream();
-#if NEWTONSOFTJSON10 // Json.NET 10's Bson package calculates the path in some exceptions differently.
-            string expectedPath = "Value";
-#else
+#if NETFX_CORE // Separate Bson package (not yet used in NETCore project) calculates the path in exceptions differently
             string expectedPath = string.Empty;
+#else
+            string expectedPath = "Value";
```

Comment:
Just to confirm, this inversion in logic was intentional?

Reply:
Yes. We have 3 `Http.Formatting` projects in src: `NetCore` for `Profile259` (which is supposed to be similar to `netstandard1.3` but is actually a superset in some ways), `NetStandard` for `netstandard2.0`, and "regular" for .NET Framework. Previously only the `NetStandard` project defined `NEWTONSOFTJSON10` and used the Json.Bson package. Now all but the `NetCore` project uses Json.Bson. That project defines `NETFX_CORE`.